widgetastic_manageiq/__init__.py

```diff
@@ -598,6 +598,54 @@ def read(self):
         return [cb for cb in self.checkboxes if cb.selected]
 
 
+class BootstrapSwitchSelect(CheckboxSelect):
+    """This view is very similar to parent CheckboxSelect view. BootstrapSwitches used instead of
+    usual Checkboxes. It can be found in the same policy's events assignment screen since
+    CFME 5.8.1.
+    """
+    BS_TEXT = '/../../following-sibling::text()[1]'
+
+    def _get_bs_description(self, bs):
+        """Returns text description of the BootstrapSwitch widget. We have to use such hack with
+        the script execution, because Selenium cannot return text of a text node itself.
+
+        Returns: str
+        """
+        return bs._label or self.browser.execute_script(
+            "{script} return xpath(null, {arg}).textContent;".format(
+                script=DynaTree.XPATH,
+                arg=quote(bs.ROOT.locator + self.BS_TEXT)
+            )).strip()
+
+    @property
+    def checkboxes(self):
+        """All bootstrap switches."""
+        return {BootstrapSwitch(self, id=el.get_attribute("id")) for el in self.browser.elements(
```

Comment:
Well, I certainly don't like it, but this can be converted later into a better solution.